models/safe/ethereum/safe_eth_balances.sql

```diff
@@ -0,0 +1,81 @@
+{{
+    config(
+        materialized='incremental',
+        alias='eth_safes_balances',
+        partition_by=['day'],
+        unique_key=['day', 'address', 'balance', 'token_address', 'token_id'],
```

Comment:
your unique keys include columns which aren't written to the target spell. please see the columns written in lines 64-66 below, only three columns. unique keys should be one or two columns only, if there are three total. otherwise, there are no columns to update in a merge statement